Tests/validate_release_notes.py

```diff
@@ -0,0 +1,92 @@
+import pip
+
+try:
+    import yaml
+except ImportError:
+    pip.main(['install', 'pyyaml'])
+    import yaml
+import json
+import sys
+import re
+from subprocess import Popen, PIPE
+
+# file types regexes
+INTEGRATION_REGEX = "(integration-).*(.yml)"
```

Comment:
we have this regex duplicate here, lets see if we can remove this duplication
I would call this script/logic from `validate_schema.py` (if `is_local` is true maybe)